src/order/compactly_generated.lean

```diff
@@ -5,6 +5,9 @@ Authors: Oliver Nash
 -/
 import order.well_founded
 import order.order_iso_nat
+import order.atoms
+import order.lattice_intervals
```

Comment:
Is this not imported by `order.atoms`?

Reply:
Looks like `data.set.finite` wasn't needed either.